upstream/CVE-2023-3955.json

```diff
@@ -53,12 +53,199 @@
                     ]
                 }
             ]
+        },
+        {
+            "package": {
+                "ecosystem": "eks",
+                "name": "k8s.io/kubelet"
+            },
+            "severity": [
+                {
+                    "type": "CVSS_V3",
+                    "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H"
+                }
+            ],
+            "ranges": [
+                {
+                    "type": "SEMVER",
+                    "events": [
+                        {
+                            "introduced": "1.27.0"
+                        },
+                        {
+                            "fixed": "1.27.4-eks-8ccc7ba"
```

Comment:
I'm wondering if this version is comparable. 8ccc7ba looks like a commit hash.

Reply:
yes its comparable but it is identified  `eks-8ccc7ba` as pre-release 

`1.27.4-eks-8ccc7ba  < 1.27.0`

Reply:
It is not our desired result, right?

Reply:
true, we need to figure out how to solve this use case